### PR TITLE
expand log access docs, add db access to ssh doc

### PR DESCRIPTION
**Pull request by tannerjfco**
## The Problem:
With drud/docker.nginx-php-fpm#40 ddev logs will only output error log, but we still want to provide some info for advanced users to access other logs.
## The Fix:
This PR updates the log section of docs to document location of access logs and php-fpm logs in the web container. It also updates the ssh section to provide instruction for accessing the db container.

These changes are intended to be the documentation component of https://github.com/drud/ddev/issues/87#issuecomment-297091699

## The Test:
Docs change. If it makes sense, great! If not, suggestions are welcome for improving.
## Automation Overview:

Docs change, no test changes.
## Related Issue Link(s):
https://github.com/drud/ddev/issues/87#issuecomment-297091699

### CODE REVIEW

**Comment by tannerjfco** [comment]
Sure, I can do that once drud/docker.nginx-php-fpm#40 is approved

**Comment by rickmanelius** [comment]
Changes look fine and dandy to me.

File: cmd/ddev/cmd/logs_test.go [hunk 1]
```diff
@@ -37,7 +37,6 @@ func TestDevLogs(t *testing.T) {
 		out, err := system.RunCommand(DdevBin, args)
 		assert.NoError(err)
 		assert.Contains(string(out), "Server started")
-		assert.Contains(string(out), "GET")
```
Comment (beeradb) [inline cmd/ddev/cmd/logs_test.go]: I wonder if we should trigger a PHP error and ensure it ends up in the log?

File: cmd/ddev/cmd/logs_test.go [hunk 2]
```diff
@@ -33,11 +37,23 @@ func TestDevLogs(t *testing.T) {
...
+		err = network.EnsureHTTPStatus(o)
+		assert.NoError(err)
+
 		args := []string{"logs"}
 		out, err := system.RunCommand(DdevBin, args)
+
 		assert.NoError(err)
 		assert.Contains(string(out), "Server started")
-		assert.Contains(string(out), "GET")
+		assert.Contains(string(out), "PHP message: PHP Stack trace:")
```
Comment (rfay) [inline cmd/ddev/cmd/logs_test.go]: Might as well add an `assert.NotContains(string(out), 'GET')` here since that's what we're getting rid of.

Comment (tannerjfco) [inline cmd/ddev/cmd/logs_test.go]: GET will still show in the error log, as the request info in the PHP error message.

Comment (rfay) [inline cmd/ddev/cmd/logs_test.go]: Might be possible to do something more explicit, but I'm fine with whatever. Good to bring this in whenever you're happy.

**Pull request merged**